pkg/pillar/types/locationconsts.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2017 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package types
+
+const (
+	appImgObj = "appImg.obj"
```

Comment:
Seems like a random subset of constants.

If you want to move files related to /var/tmp and /persist I'd move all of them. But as we talked, the pathnames are likely to become local to each microservice as we disaggregate into separate service VMs/containers.

Reply:
Yeah.. wanted to move the others in a separate change.. i only moved ones needed for the current change - instead of replicating them.

Yeah.. still thinking about how to handle the separate VMs / containers.. But the thinking is they will still have a very parameterized structure.. 

have another cleanup change in progress to take care of the rest of common constants..